index.js

```diff
@@ -1,3 +1,7 @@
+const { setTraceFunction } = require('hypertrace')
+let tracesCount = 0
+setTraceFunction(() => { tracesCount++ })
+process.on('exit', () => console.log(`trace function called ${tracesCount} times`))
```

Comment:
Maybe only console.log if `tracesCount > 0`? So we don't have that log unless we're tracing